humble/distribution.yaml

```diff
@@ -9763,6 +9763,17 @@ repositories:
       url: https://github.com/ros2/system_tests.git
       version: humble
     status: developed
+  talos_moveit_config:
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/pal-gbp/talos_moveit_config-release.git
```

Comment:
This is OK to release into Humble, but if you ever want to release this into Rolling, it would need to be moved in to the `ros2-gbp` repository.